core/src/test/scala/integration/kafka/admin/RemoteTopicCrudTest.scala

```diff
@@ -441,7 +441,36 @@ class RemoteTopicCrudTest extends IntegrationTestHarness {
           AlterConfigOp.OpType.SET),
       ))
     assertThrowsException(classOf[InvalidConfigurationException],
-      () => admin.incrementalAlterConfigs(configs).all().get(), "Disabling remote storage feature on the topic level is not supported.")
+      () => admin.incrementalAlterConfigs(configs).all().get(), "It is invalid to disable remote storage without deleting remote data. " +
+        "If you want to keep the remote data and turn to read only, please set `remote.storage.enable=true,remote.log.copy.disable=true`. " +
+        "If you want to disable remote storage and delete all remote data, please set `remote.storage.enable=false,remote.log.delete.on.disable=true`.")
+  }
+
+  @ParameterizedTest
+  @ValueSource(strings = Array("kraft"))
+  def testUpdateTopicConfigWithDisablingRemoteStorageWithDeleteOnDisable(quorum: String): Unit = {
```

Comment:
We already have `testCreateRemoteTopicWithCopyDisabledAndDeleteOnDisable` to test the same thing. I think this is not needed.

Reply:
@showuon, `testCreateRemoteTopicWithCopyDisabledAndDeleteOnDisable` is about creating topic. This one is about incremental config change. I think the scenario is different.

Reply:
Ah, you're right. Thanks for the explanation.